x/auth/keeper/keeper.go

```diff
@@ -274,3 +283,17 @@ func (ak AccountKeeper) getBech32Prefix() (string, error) {
 
 	return bech32Codec.bech32Prefix, nil
 }
+
+// SetParams sets the auth module's parameters.
+func (ak AccountKeeper) SetParams(ctx context.Context, params types.Params) error {
+	if err := params.Validate(); err != nil {
+		return err
+	}
+	return ak.ParamsState.Set(ctx, params)
+}
+
+// GetParams gets the auth module's parameters.
+func (ak AccountKeeper) GetParams(ctx context.Context) (params types.Params) {
+	params, _ = ak.ParamsState.Get(ctx)
```

Comment:
The previous `GetParams` was panicking, we probably should here too.

Reply:
right, modified the API to panic only in case error is not a `NotFound` one, this should retain the old behaviour.